download/index.md

```diff
@@ -10,6 +10,7 @@ other_releases: [
   ["maintenance_version", "Last 2.10.x maintenance release", 2.10.7, "November 9, 2017"]
 ]
 requirements: "This Scala software distribution can be installed on any Unix-like or Windows system. It requires Java 8 or later, available <a href='http://www.java.com/'>here</a>."
+license: <a href="https://www.scala-lang.org/license/">Apache License, Version 2.0</a>
```

Comment:
Should the link read `"!SITE_BASE_URL!/license/"` instead?

Reply:
Ah, scratch that. Things have changed since the initial PR https://github.com/scala/scala-lang/pull/1019.